TYPO3.Flow/Classes/TYPO3/Flow/Core/Bootstrap.php

```diff
@@ -422,6 +422,9 @@ protected function resolveRequestHandler()
                 $suitableRequestHandlers[$priority] = $requestHandler;
             }
         }
+        if (empty($suitableRequestHandlers)) {
+            throw new FlowException('No suitable request handler could be found for the current request. This is most likely a setup-problem, so please check your package.json and/or try removing Configuration/PackageStates.php', 1464882543);
```

Comment:
Hm, there's indeed no import of the namespace here - I was damn sure the class was imported, actually I always auto-complete classes in IntelliJ to make sure I don't forget. Only thing I could imagine is I first added this change on master, then rebased to 2.3 and didn't notice that 2.3 didn't import the FlowException yet. Also tests didn't break because there is no test that simulates an unresolved RequestHandler. Argh, having a bad commit streak this week :(